seurat-read.R

```diff
@@ -122,7 +122,10 @@ if (! is.na(opt$data_dir)){
 
 cell_metadata<-NULL
 if ( ! is.null(opt$metadata) ) {
-  cell_metadata<-read.table(opt$metadata,header = TRUE, sep="\t", row.names = 1)
+  cell_metadata<-read.table(opt$metadata, 
+			    header = TRUE, sep="\t",
+                            row.names = 1, check.names = FALSE, 
+                            stringsAsFactors = FALSE)
```

Comment:
Sorry, I know I'm being anal- having spent hours squinting at code where people have mixed tabs and spaces, the following suggestion strips out your tabs:

```suggestion
  cell_metadata<-read.table(opt$metadata, 
                            header = TRUE, sep="\t",
                            row.names = 1, check.names = FALSE, 
                            stringsAsFactors = FALSE)
```